bin/i18n/resources/dashboard/shared_functions.rb

```diff
@@ -10,18 +12,11 @@ module SharedFunctions
         I18N_SOURCE_FILE_PATH = File.join(I18n::Resources::Dashboard::I18N_SOURCE_DIR_PATH, FILE_NAME).freeze
 
         def self.sync_in
-          puts 'Preparing shared functions'
-
-          # TODO: Refactor shared_functions request and data collection
-          #   1. Select data in batches of 1k records
-          #   2. Optimize data collection
```

Comment:
Do we still want to do this? If so, I would keep the comment in the new sync_in module.

Reply:
No, I fixed this